apps/cli/test/credentialsStorage.test.ts

```diff
@@ -0,0 +1,71 @@
+import { fs, type DirectoryJSON, vol } from "memfs";
+import { beforeEach, describe, expect, it, vi } from "vitest";
+
+// Mock all fs promises functions
+vi.mock("node:fs/promises", () => fs.promises);
+
+// Use a mock homedir in order to reliably have a memfs home directory
+vi.mock("node:os", () => ({ homedir: () => "/home/codemod-test/" }));
+
+import {
+  CredentialsStorage,
+  CredentialsStorageType,
+} from "../src/credentials-storage.js";
+
+describe("CredentialsStorage", () => {
+  const testService = "codemod.com-test";
+  const testAccount = CredentialsStorageType.ACCOUNT;
+  const testPassword = "test-password";
+
+  describe("when .codemod directory does not exist", () => {
+    beforeEach(() => {
+      vol.reset();
```

Comment:
This before each block is repeated on each test. Would be happy to move to enclosing scope.